### PR TITLE
[v5] Remove nightwatch!

### LINKED ISSUE #165: Warning about requiring a peer of nightwatch despite terra-toolkit warning against using nightwatch
# Bug Report

## Description
After installing terra and dependencies, it was discovered that nightwatch introduced some vulnerabilities. [After seeing this warning in terra-toolkit's very own code](https://github.com/cerner/terra-toolkit/blob/bde3c9e047d6bfc18c2388ea7dcd3afc182e98f7/src/nightwatch/nightwatch.config.js#L14) we removed nightwatch and installed webdriver.io. However, running npm install at this point will give a warning that terra toolkit requires a peer of nightwatch.

## Steps to Reproduce
1. Install terra/terra-toolkit
2. See vulnerabilities. Remove nightwatch and replace with webdriver.io
3. npm install
4. See warning

## Additional Context / Screenshots
image

## Expected Behavior
No warnings for removing a package that terra-toolkit suggests we remove.

## Possible Solution
Not sure where that warning appears but perhaps check for webdriver.io and not have this warning if webdriver.io is installed.

## Environment

* Node/npm Version: Node >=4.x/npm 6.4.1
* Webpack Version: webpack 4.16.5
* Operating System and version (desktop or mobile): Windows

### LINKED ISSUE #172: Remove nightwatch dependency
# Feature Request

## Description
Remove nightwatch dependency

## Additional Context / Screenshots
We've deprecated the use of nightwatch and recommend users switch over to using webdriver.io tests. In our next major version bump of terra-toolkit, we plan on removing this dependency.

### PR DESCRIPTION
**Pull request by emilyrohrbough**
### Summary
I've been wanting to do this for a year and a half! 😄 

Resolves https://github.com/cerner/terra-toolkit/issues/165 
Resolves https://github.com/cerner/terra-toolkit/issues/172

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -110,9 +108,6 @@
     "webpack-dev-server": "^3.1.11",
     "webpack-merge": "^4.2.1"
   },
-  "peerDependencies": {
-    "nightwatch": "^0.9.12"
-  },
   "optionalDependencies": {
     "wdio-mocha-framework": "^0.5.11"
   },
```
Comment (StephenEsser) [inline package.json:111]: Nightwatch is still sneaking around in a few places, do we want to remove the devDependency on it?

Comment (emilyrohrbough) [reply package.json:111]: I cannot believe I missed that! 🤦‍♀️

**Review by mjhenkes — APPROVED** [review]

**Review by bjankord — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -129,8 +124,7 @@
     "eslint": "^5.12.0",
     "eslint-config-terra": "^2.2.0",
     "html-webpack-plugin": "^3.1.0",
-    "jest": "^23.1.0",
-    "nightwatch": "^0.9.12",
+    "jest": "^23.1.0"
```
Comment (StephenEsser) [inline package.json:126]: The famous missing comma of destruction.

**Review by yuderekyu — APPROVED** [review]

File: .npmignore [hunk 3]
```diff
@@ -7,7 +7,6 @@ tests
 errorScreenshots
```
Comment (StephenEsser) [inline .npmignore:7]: Should we remove these?

https://github.com/cerner/terra-toolkit/blob/toolkit-v5-remove-nightwatch/tests/fixtures/nightwatch.html

https://github.com/cerner/terra-toolkit/blob/toolkit-v5-remove-nightwatch/tests/test.config.js#L10

**Review by StephenEsser — APPROVED** [review]

**Review by benbcai — APPROVED** [review]

**Pull request merged**